packages/driver/src/cypress/browser.coffee

```diff
@@ -1,32 +1,19 @@
 _ = require("lodash")
 $utils = require("./utils")
 
-browserFamilyMap = {
-  chrome: "chrome"
-  canary: "chrome"
-  chromium: "chrome"
-  electron: "chrome"
+isBrowser = (config, obj='') ->
```

Comment:
since we are supporting `--browser canary` to be backwards compatible... should we update `Cypress.isBrowser()` to also support that, or is that a terrible idea, i can't tell...

Reply:
nah, this is a new API, so it doesn't need to be backwards compatible. i'm ok with this and `--browser` having different logic